brain_games/scripts/brain_even.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import random
+
+from prompt import string
+
+from .brain_games import main as brain_games
+
+
+def main() -> None:
+    """
+    Asks a question if the random number is even or not.
+    Game is completed in case of three correct answers in row
+    """
+    count = 0
+    name = brain_games()
+    while count < 3:
+        print('Answer "yes" if the number is even, otherwise answer "no".')
+        num = random.randint(0, 100)
+        print(f"Question: {num}")
+        answer = string("Your answer: ")
+        if (num % 2 == 0 and answer == "yes") or (num % 2 != 0 and answer == "no"):
+            print("Correct!")
+            count += 1
+
+        elif num % 2 != 0 and answer == "yes":
+            correct_answer = "yes" if answer == "no" else "no"
+            print(
+                f"'{answer}' is wrong answer ;(. Correct answer was '{correct_answer}'."
+                f"\nLet's try again, {name}!'"
+            )
+            count = 0
+
+        else:
+            print(
+                f"'{answer}' is wrong answer ;(. Correct answer was 'yes'."
+                f"\nLet's try again, {name}!'"
+            )
+            count = 0
+
+    print(f"Congratulations, {name}!")
```

Comment:
game logic should be stored in a separate place  
in general, scripts should just call functions from other modules